clients/admin-ui/src/features/config-wizard/AuthenticateAwsForm.tsx

```diff
@@ -110,101 +128,193 @@ const AuthenticateAwsForm = () => {
+                If you cancel scanning, the scanner will stop and no systems
+                will be returned.
+              </Text>
+              <Text color="gray.500" mb={3}>
+                Are you sure you want to cancel?
+              </Text>
+            </AlertDialogBody>
+
+            <AlertDialogFooter>
+              <Button onClick={() => handleCancel()} variant="outline">
+                Yes, Cancel
               </Button>
               <Button
-                type="submit"
-                variant="primary"
-                isDisabled={!dirty || !isValid}
-                isLoading={isLoading}
-                data-testid="submit-btn"
+                colorScheme="primary"
+                ref={cancelRef}
+                onClick={onClose}
+                ml={3}
               >
-                Save and Continue
+                No, Continue Scanning
               </Button>
-            </HStack>
-          </Stack>
-        </Form>
-      )}
-    </Formik>
+            </AlertDialogFooter>
+          </AlertDialogContent>
+        </AlertDialogOverlay>
+      </AlertDialog>
+      <Formik
+        initialValues={initialValues}
+        validationSchema={ValidationSchema}
+        onSubmit={handleSubmit}
+      >
+        {({ isValid, isSubmitting, dirty }) => (
+          <Form data-testid="authenticate-aws-form">
+            <Stack spacing={10}>
+              {isSubmitting ? (
+                <Text
+                  alignItems="center"
+                  as="b"
+                  color="gray.900"
+                  display="flex"
+                  fontSize="xl"
+                >
+                  System scanning in progress{" "}
+                  <CloseButton
+                    data-testid="close-scan-in-progress"
+                    display="inline-block"
+                    onClick={onOpen}
+                  />
+                </Text>
+              ) : null}
+              {!systemResultsNumber && isSubmitting ? (
+                <Stack alignItems="center">
+                  <Spinner
+                    thickness="4px"
+                    speed="0.65s"
+                    emptyColor="gray.200"
+                    color="gray.500"
+                    size="md"
+                  />
+                </Stack>
+              ) : null}
+              {systemResultsNumber && !isSubmitting ? (
+                <Stack alignItems="baseline">
+                  <Text as="b">
+                    <GreenCheckCircleIcon />
+                    {systemResultsNumber} systems identified
+                  </Text>
+                  <Button
+                    backgroundColor="transparent"
+                    color="purple.500"
+                    fontWeight="400"
+                    data-testid="view-scan-results-button"
+                    onClick={() => dispatch(changeStep())}
+                  >
+                    View results <ChevronRightIcon color="purple.500" />
```

Comment:
The designs show this step which shows the number of systems found/scanned, and then a separate click to view the results, but I'm not 100% convinced that this extra click is needed, at least at the moment. For now, I put it here since it's part of the designs, but perhaps it's a question that should be surfaced to design.